docs/tutorials/cargo_tests/baked/build.rs

```diff
@@ -0,0 +1,27 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use icu::locid::langid;
+use icu_datagen::CldrLocaleSubset;
+use icu_datagen::SourceData;
+use icu_provider::marker::KeyedDataMarker;
+use std::path::PathBuf;
+
+fn main() {
+    icu_datagen::datagen(
+        Some(&[langid!("ru")]),
+        &[icu::plurals::provider::CardinalV1Marker::KEY],
+        &SourceData::default()
+            // Note: We use "latest" to stay fresh with the latest CLDR data.
+            .with_cldr_for_tag("latest", CldrLocaleSubset::Modern)
+            .expect("Source data should download successfully"),
+        vec![icu_datagen::Out::Module {
+            mod_directory: PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("baked_data"),
+            pretty: true,
+            insert_feature_gates: false,
```

Comment:
Maybe add comments for these?

Reply:
It's hard to write comments that are as useful as the docs comments on Out::Module; they lack context, and good comments here would be just duplicating the docs. But I can add a link to the docs page.